main.js

```diff
@@ -162,10 +162,134 @@ const inventory = [
+        revenue += (inventoryArray[inventoryKey].originalStock - inventoryArray[inventoryKey].sold) * inventoryArray[inventoryKey].price;
+    }
+    return revenue;
+}
+
+function calculateCurrentRevenue(inventoryArray) {
+    let currentRevenue = 0;
+    for (const inventoryKey in inventoryArray) {
+        currentRevenue += inventoryArray[inventoryKey].sold * inventoryArray[inventoryKey].price;
+    }
+    return currentRevenue;
+}
+
+const revenueTarget = document.getElementById("revenueTarget");
+revenueTarget.textContent += calculateRevenueTarget(inventory);
+revenueTarget.style.color = "blue";
+
+const sellingStock = document.getElementById("stockToSell");
+sellingStock.textContent += calculateAmountToSell(inventory);
+sellingStock.style.color = "red";
+
+const currentRevenue = document.getElementById("currentRevenue");
+currentRevenue.textContent += calculateCurrentRevenue(inventory);
+currentRevenue.style.color = "green";
+
+const typeOfTVOne = document.getElementById("typeOfTVOne");
+typeOfTVOne.textContent += inventory[1].type;
+
+const typeOfTVTwo = document.getElementById("typeOfTVTwo");
+typeOfTVTwo.textContent += inventory[2].type;
+
+const revenueContainer = document.getElementById("revenueContainer");
+
+function formatInventoryArray(inventory) {
+    for (let i = 0; i < inventory.length; i++) {
+        const tvFormatted = document.createElement("p");
+        tvFormatted.setAttribute("Id", "tvFormatted" + i);
+        tvFormatted.setAttribute('style', 'white-space: pre;');
+        tvFormatted.textContent = formatTVName(inventory[i]) + " \n" +
+            formatPrice(inventory[i].price) + " \n" +
+            formatTVSize(inventory[i].availableSizes);
+        revenueContainer.appendChild(tvFormatted);
+    }
+}
+
+formatInventoryArray(inventory);
+
+function formatTVName(television) {
+    return television.brand + " " + television.type + " - " + television.name + " \r";
+}
+
+function recalculateScreenSizeToCM(sizeInInches) {
+    return (sizeInInches / 0.39370).toFixed(0);
+}
+
+function formatTVSize(televisionSize) {
+    let screenSizeString = "";
+    for (const televisionSizeKey in televisionSize) {
+        if (!(televisionSize[televisionSizeKey] === televisionSize[televisionSize.length - 1])) {
+            screenSizeString += televisionSize[televisionSizeKey] + " inches " + "(" + recalculateScreenSizeToCM(televisionSize[televisionSizeKey]) + " cm) | ";
+        } else {
+            screenSizeString += televisionSize[televisionSizeKey] + " inches " + "(" + recalculateScreenSizeToCM(televisionSize[televisionSizeKey]) + " cm)";
+        }
+    }
+    return screenSizeString;
+}
+
+function clearElements(inventory) {
+    for (let i = 0; i < inventory.length; i++) {
+        let child = document.getElementById("tvFormatted" + i);
+        revenueContainer.removeChild(child);
+    }
+}
+
+let priceSort = document.getElementById("priceSort")
+priceSort.addEventListener("click", function sortByPrice() {
```

Comment:
Ik denk dat je dit zelf al weet, maar `function sortByPrice` is in deze overbodig. Het had dus dit kunnen zijn:
```priceSort.addEventListener("click", () => {```